src/openforms/contrib/auth_oidc/plugin.py

```diff
@@ -104,6 +104,13 @@ def before_process_claims(self, config: BaseConfig, claims: JSONObject):
     def strict_mode(self, request: HttpRequest) -> bool:
         return False
 
+    def sensitive_claims(self, options: OptionsT, claims: JSONObject) -> list[str]:
```

Comment:
```suggestion
    def sensitive_claims(self, options: OptionsT, claims: JSONObject) -> Sequence[str]:
```
if possible, specifying the interface rather than the implementation is a bit more robust/flexible/future proof

in this case, I don't think the ordering is even relevant, so you may even want to use `Collection` instead